app/controllers/onboarding_controller.rb

```diff
@@ -131,7 +133,7 @@ def jwt_param
     params.require(:jwt)
   end
 
-  def organization
+  def organization_from_token
```

Comment:
Can't we remove this now? We should no longer encode the organization id in the jwt, but instead use the organization from the invites route. Or do we keep this for some time to not invalidate existing links?

Reply:
Ah! I didn't know that. Then I will stop #1960 from getting merged.